azure-stack/hci/upgrade/install-enable-network-atc.md

```diff
@@ -109,13 +109,9 @@ Set-service -Name NetworkATC -StartupType Automatic
 
 ### Step 6: Add the Network ATC intent
 
-There are various intents that you can add. Identify the intent or intents you'd like using the examples in the next section.
+There are various intents that you can add. Identify the intent or intents you would like using the examples in the next section.
```

Comment:
@helmste - I think the conjunction here is fine and it's recommended in our style guide. We can leave it as is.

Reply:
@ronmiab - that's fine to leave if you prefer.  I usually see these spelled out to avoid confusion.